frontend/src/components/SimpleDropdownSelect.tsx

```diff
@@ -1,42 +1,64 @@
 import * as React from 'react';
 import { Dropdown, DropdownItem, DropdownToggle } from '@patternfly/react-core';
+import './SimpleDropdownSelect.scss';
 
 type SimpleDropdownProps = {
-  options: { key: string; label: React.ReactNode }[];
+  options: {
+    key: string;
+    label: React.ReactNode;
+    description?: React.ReactNode;
+    selectedLabel?: React.ReactNode;
+    isPlaceholder?: boolean;
+  }[];
   value: string;
   placeholder?: string;
-  onChange: (key: string) => void;
+  onChange: (key: string, isPlaceholder: boolean) => void;
+  isFullWidth?: boolean;
```

Comment:
This is a really good case for storyboard testing.
Adding tests for our custom components and document them with storyboard might be really useful for when we add new props and so.

Reply:
true. I would like to fully add tests to this feature after i get it into incubation.